.vitepress/errors.data.ts

```diff
@@ -0,0 +1,8 @@
+import { defineLoader } from 'vitepress'
+import { errorMessages } from '@vue/compiler-core'
+
+export default defineLoader({
+  load() {
+    return { errorMessages }
```

Comment:
53 (or the largest in the keys) is an extension point which is empty, we should filter out empty keys here